src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navbar: FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => {
+    return cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink to="/" className={getLinkClass}>
+            Home
+          </NavLink>
+
+          <NavLink to="/people" aria-current="page" className={getLinkClass}>
```

Comment:
[TASK] Keep search params when navigating within the People page (when selecting a person or **clicking the People link**).